modules/rational-ui.el

```diff
@@ -127,5 +127,11 @@ Used as hook for modes which should not display line numebrs."
 				                     recenter-top-bottom other-window))
   (advice-add command :after #'pulse-line))
 
+;; Italic comments and keywords in server-mode
+(set-face-attribute 'font-lock-comment-face nil
+  :slant 'italic)
+(set-face-attribute 'font-lock-keyword-face nil
+  :slant 'italic)
+
```

Comment:
I would suggest removing the Italic comments section from this PR.

Italicizing comments is something often handled by a theme. It does not seem right that we would override the user's theme preference by forcing italic comments here.

Reply:
How about we give user a choice. Whether they want to use italic comments or not?

Reply:
I wouldn't add this here, would be something to put in config.el or inherit from the theme